components/arm/arm.go

```diff
@@ -104,6 +104,8 @@ type Arm interface {
 	// Stop stops the arm. It is assumed the arm stops immediately.
 	Stop(ctx context.Context, extra map[string]interface{}) error
 
+	AllInputs(ctx context.Context, goals [][]referenceframe.Input) error
```

Comment:
I love the concept; I'm not sure I love the `AllInputs` name. Could we workshop that a bit?

Reply:
Agree.  Maybe something like `GoToWaypoints` `GoToPlan` `ExecutePlan` ?  We talked Friday briefly about a nomenclature alignment, this could be a chance for us to codify the concept of a "Plan" as a list of Inputs

Another approach could be switching to `GoToInput` vs `GoToInputs`